usaon_vta_survey/__init__.py

```diff
@@ -85,11 +94,14 @@ def before_request():
     app.register_blueprint(login_bp)
     app.register_blueprint(logout_bp)
     app.register_blueprint(blueprint, url_prefix="/google_oauth")
```

Comment:
Hate to nitpick on one more... should this be `google_bp` or `google_login_bp` for consistency?